brayovsky/pivotaly
# Add refresh button to backlog

#### What does this PR do? / Fixes issue
Adds a button to refresh backlog.
Solves issue #95 

#### How should this be manually tested?
Open current/backlog viewlet. A refresh button should be available on the navigation menu.

#### Any background context you want to provide?
N/A

#### Screenshots (if appropriate)
<img width="280" alt="Screen Shot 2020-02-25 at 18 26 23" src="https://user-images.githubusercontent.com/19430730/75261610-5e148680-57fc-11ea-858a-a79e43b3daba.png">

#### Questions:
